src/ui/OverlayController.tsx

```diff
@@ -82,19 +99,55 @@ export class OverlayController extends React.Component<IControllerProps, void> {
     this.props.wrapperElement.classList.toggle(CSS.RMX_VISIBLE);
   }
 
+  /** Toggles the share menu visibility. */
+  toggleShareMenu() {
```

Comment:
Looking at this again, I'm guessing you took the wrapping function approach because if you might have been running into issues with `this` being rebound.

You can solve that by making this a property that points at a lambda, rather than a method:

```typescript
toggleShareMenu = () => {
  // ...
}
```